src/js/components/atoms/c-image/c-image.models.ts

```diff
@@ -1,4 +1,5 @@
-import { Image } from '@ultimaker/ultimaker.com-model-definitions/dist/atoms/media/Image';
+import { FocusArea, Image } from '@ultimaker/ultimaker.com-model-definitions/dist/atoms/media/Image';
+export { FocusArea } from '@ultimaker/ultimaker.com-model-definitions/dist/atoms/media/Image';
```

Comment:
Huh? Why, Import where needed?

Reply:
I have the others still in the c-image models. For now it is nice to keep hem in one place. Maybe in a while the all come from the global models ;-)